lib/internal/dns/utils.js

```diff
@@ -94,7 +95,7 @@ class Resolver {
         ipVersion = isIP(hostIP);
 
         if (ipVersion !== 0) {
-          return newSet.push([ipVersion, hostIP, parseInt(port)]);
+          return newSet.push([ipVersion, hostIP, NumberParseInt(port)]);
```

Comment:
```suggestion
          return ArrayPrototypePush(newSet, [ipVersion, hostIP, NumberParseInt(port)]);
```
(also in a couple other places in this file)